tests/Functional/src/com/microsoft/aad/adal/test/AuthenticationContextTest.java

```diff
@@ -1159,7 +1158,7 @@ public void testScenario_LoginHint_IdToken_Different() throws InterruptedExcepti
         // Same call with correct upn will return from cache
         final CountDownLatch signalCallback2 = new CountDownLatch(1);
         callback.mSignal = signalCallback2;
-        context.acquireToken(null, "resource", "clientid", "redirectUri", idtoken.upn, callback);
+        context.acquireToken(testActivity, "resource", "clientid", "redirectUri", idtoken.upn, callback);
```

Comment:
:) Sure. I saw the tests failed and then fixed them. Will let you know next time. Oh, by the way, i'm looking at the tests failure you mentioned in the email. That should be the service we are hitting for testing purpose, but they're no longer working I think. I'm going to ask around who set them up and who is maintaining them.